build.sbt

```diff
@@ -95,6 +95,14 @@ inThisBuild(
   )
 )
 
+onLoad.in(Global) ~= { old =>
+  import scala.sys.process._
+  require(
+    "yarn install".! == 0,
+    "command failed: yarn install. Is yarn installed? https://yarnpkg.com/en/docs/install"
```

Comment:
Since yarn is not exactly standard in Scala builds, maybe it's worth adding some context here in the error message?

Reply:
Also, we're not really using anything node-specific here.
What about replacing husky with a simpler script that just installs the git hooks we need on load?